### ISSUE TITLE
call_pparallel without filtering produces vcf.gz unconditionally

### ISSUE THREAD

**Issue by wm75** [CSB5/lofreq#85]
If the final ouput destination is a file, the script simply copies the concatenated data even if the final file name does not end in.gz:

https://github.com/CSB5/lofreq/blob/f2c00dc3345f9be2a5f400d171cfe942e335cf5f/src/scripts/lofreq2_call_pparallel.py#L722

**Comment by wm75** [comment]
The alternative code path, with final output going to stdout:
https://github.com/CSB5/lofreq/blob/f2c00dc3345f9be2a5f400d171cfe942e335cf5f/src/scripts/lofreq2_call_pparallel.py#L713
 fails under Python 3 because gzip reads bytes, but stdout is using text mode.

**Comment by andreas-wilm** [comment]
Thanks Wolfgang. To address the first problem, I'll change the docs to make it clear that the output is always compressed vcf. I can't reproduce the second problem, but am tempted to simply not allow stdout.

**Comment by wm75** [comment]
Hi Andreas! If you're interested in PRs fixing those issues, I may also be able to provide those after the holidays. We recently wrapped most of lofreq's functionality for Galaxy (should become available on usegalaxy.eu over Christmas and on usegalaxy.org shortly after) and that's when we came across these issues. Since we're planning to use lofreq in a few Galaxy workflows, it should be possible to also contribute back, a bit at least, to your upstream code :smile:
